Categories/Category/Construction/Properties/EilenbergMoore.agda

```diff
@@ -0,0 +1,103 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Construction.Properties.EilenbergMoore where
+
+open import Level
+import Relation.Binary.PropositionalEquality as ≡
+
+open import Categories.Adjoint
+open import Categories.Adjoint.Properties
+open import Categories.Category
+open import Categories.Functor using (Functor; _∘F_)
+open import Categories.Functor.Equivalence
+open import Categories.Monad
+
+open import Categories.NaturalTransformation renaming (id to idN)
+
+open import Categories.Adjoint.Construction.EilenbergMoore
+open import Categories.Category.Construction.EilenbergMoore
+
+private
+  variable
+    o ℓ e : Level
+    𝒞 𝒟 : Category o ℓ e
+
+open NaturalTransformation
+
+module _ {F : Functor 𝒞 𝒟} {G : Functor 𝒟 𝒞} (adjoint : Adjoint F G) where
+  T : Monad 𝒞
+  T = adjoint⇒monad adjoint
+
+  𝒞ᵀ : Category _ _ _
+  𝒞ᵀ = EilenbergMoore T
+
+  module 𝒞 = Category 𝒞
+  module 𝒟 = Category 𝒟
+  module 𝒞ᵀ = Category 𝒞ᵀ
+
+  module T = Monad T
+  module F = Functor F
+  module G = Functor G
+
+  open Adjoint adjoint
+
+  open Functor F
+  open Functor G renaming (F₀ to G₀; F₁ to G₁)
```

Comment:
the usual style now is to avoid renaming, but rather use `module`. then the code is much cleaner and more predictable: https://github.com/agda/agda-categories/issues/103#issuecomment-572268282

e.g.

```agda
module G = Functor G
-- later on
G.F₀
```